cbmc/proofs/Makefile.common

```diff
@@ -20,20 +20,22 @@ VIEWER ?= cbmc-viewer
 # Build goto binaries with options taken from top-level cmake
 
 INC += \
-	-I$(MQTT)/libraries/standard/mqtt/include \
+	-I$(MQTT)/cbmc/proofs \
 	-I$(MQTT)/demos \
-	-I$(MQTT)/libraries/platform/.. \
+	-I$(MQTT)/libraries/ \
 	-I$(MQTT)/libraries/platform \
-	-I$(MQTT)/ports/common/include \
 	-I$(MQTT)/libraries/standard/common/include \
+	-I$(MQTT)/libraries/standard/mqtt/include \
 	-I$(MQTT)/libraries/standard/mqtt/src \
-	-I$(MQTT)/cbmc/proofs \
+	-I$(MQTT)/libraries/standard/mqtt/src/private \
+	-I$(MQTT)/ports/common/include \
```

Comment:
Is this just sorting?  And one .. cleanup?

Reply:
Yes, it is. Just sorting and one clean up.